### DISCUSSION TITLE
Support Traditional Chinese (繁體中文) translation for GitHub Mobile.

### DISCUSSION THREAD

**Discussion by hwhsu1231** [community/community#43001] [Mobile]
### Select Topic Area

Product Feedback

### Body

Support Traditional Chinese (繁體中文) translation for GitHub Mobile.

### Screenshots

![image](https://user-images.githubusercontent.com/71438617/210150142-9c6195df-7d5a-44ed-a6ea-5a70c3d3b318.png)

**Comment by ppap555120** [comment]
> ### Select Topic Area
> 
> 
> 
> Product Feedback
> 
> 
> 
> ### Body
> 
> 
> 
> Support Traditional Chinese (繁體中文) translation for GitHub Mobile.
> 
> 
> 
> ### Screenshots
> 
> 
> 
> ![image](https://user-images.githubusercontent.com/71438617/210150142-9c6195df-7d5a-44ed-a6ea-5a70c3d3b318.png)
> 
> 
> 
>

**Comment by www0617com** [comment]
> ### Select Topic Area
> 
> 
> 
> Product Feedback
> 
> 
> 
> ### Body
> 
> 
> 
> Support Traditional Chinese (繁體中文) translation for GitHub Mobile.
> 
> 
> 
> ### Screenshots
> 
> 
> 
> ![image](https://user-images.githubusercontent.com/71438617/210150142-9c6195df-7d5a-44ed-a6ea-5a70c3d3b318.png)
> 
> 
> 
>

**Comment by ferntsaiandyu-max** [comment]
> ### Select Topic Area
> 
> 
> 
> Product Feedback
> 
> 
> 
> ### Body
> 
> 
> 
> Support Traditional Chinese (繁體中文) translation for GitHub Mobile.
> 
> 
> 
> ### Screenshots
> 
> 
> 
> ![image](https://user-images.githubusercontent.com/71438617/210150142-9c6195df-7d5a-44ed-a6ea-5a70c3d3b318.png)
> 
> 
> 
>
